Scripts/Player_Scripts/player.gd

```diff
@@ -1,22 +1,33 @@
 extends CharacterBody3D
 
 
```

Comment:
variables should be grouped according to style guide example

Constants
Exports
Object references
Runtime variables